requirements-base.txt

```diff
@@ -55,6 +55,7 @@ snuba-sdk>=0.0.26,<1.0.0
 simplejson==3.17.2
 statsd==3.3
 structlog==21.1.0
+svglib==1.1.0
```

Comment:
I've never added a library before but I'm going off of https://develop.sentry.dev/python-dependencies/#rules-on-dependencies

This is the library:
https://pypi.org/project/svglib/

PIL/Pillow doesn't handle SVG files, so I needed something that does as we use SVGs for SentryApp logos.

Reply:
^^ @getsentry/owners-python-build